### PR TITLE
display 'Deployments' (#66)

### LINKED ISSUE #66: subtask for #35: display Deployments
here's how vscode is displaying them:
image

depends on #35 

here's a yaml to create a deployment:
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: sise-deploy
spec:
  replicas: 2
  selector:
    matchLabels:
      app: sise
  template:
    metadata:
      labels:
        app: sise
    spec:
      containers:
      - name: sise
        image: quay.io/openshiftlabs/simpleservice:0.5.0
        ports:
        - containerPort: 9876
        env:
        - name: SIMPLE_SERVICE_VERSION
          value: "0.9"
```

### PR DESCRIPTION
**Pull request by adietish**
fixes #66

### CODE REVIEW

**Comment by adietish** [comment]
Here's how deployments are displayed with this PR:
image

In comparison, here's vscode:
image

**Comment by adietish** [comment]
@jeffmaury please review

**Comment by jeffmaury** [comment]
Not working for me. I have install OpenShift Pipelines on CRC and this is what I get:
image

VSCode:
image

File: src/main/kotlin/org/jboss/tools/intellij/kubernetes/model/resource/Filters.kt [hunk 1]
```diff
@@ -42,3 +43,10 @@ class PodForService(private val service: Service) : Predicate<Pod> {
 		return service.spec.selector.all { pod.metadata.labels.entries.contains(it) }
 	}
 }
+
+class PodForDeployment(private val deployment: Deployment) : Predicate<Pod> {
+
+	override fun test(pod: Pod): Boolean {
+		return deployment.metadata.labels.all { pod.metadata.labels.entries.contains(it) }
```
Comment (jeffmaury) [inline src/main/kotlin/org/jboss/tools/intellij/kubernetes/model/resource/Filters.kt:50]: This is wrong: you should use the selector from the Deployment spec

**Comment by adietish** [comment]
@jeffmaury thx for spotting. I changed the predicate to:
```
class PodForDeployment(private val deployment: Deployment) : Predicate<Pod> {

	override fun test(pod: Pod): Boolean {
		return deployment.spec.selector.matchLabels.all { pod.metadata.labels.entries.contains(it) }
	}
}
```
here's what I get with this change:
image

**Pull request merged**